config/config.go

```diff
@@ -1,16 +1,21 @@
 package config
 
-/*
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
+	"io"
 	"os"
+	"path/filepath"
+
+	"github.com/facebookgo/atomicfile"
 )
-*/
+
+var ErrNotInitialized = errors.New("node not initialized, please run configure")
 
 type Config struct {
```

Comment:
Rename this to `Node` and move the struct and its methods into its own file. Then create an empty `config` struct in this file - it will have all the helper methods, load, save, etc. It can then be embedded in `Shuttle` and `Estuary` config structs

Reply:
Agree, I was just being lazy here.

Reply:
For now, I have renamed the Config struct to NodeConfig (to keep things consistent), and moved it into a new file node.go